projects/canopy/src/lib/header/header.component.ts

```diff
@@ -77,18 +86,53 @@ export class LgHeaderComponent implements AfterContentInit, OnDestroy {
         : 'lg-header-logo__second-img';
     });
 
+    // When tabbing out of the logo on sm devices, focus the account menu not the primary nav if the primary nav is open
+    this.headerLogoTabOutSubscription = this.headerLogos.last.tabbedOut
```

Comment:
This is fine, but I would have done it by setting up a subscription array instead of having 5 different subscriptions variables. Then you can just do a `forEach` in the `ngOnDestroy`.